include/seqan3/core/platform.hpp

```diff
@@ -185,6 +185,17 @@
 #   define SEQAN3_DEPRECATED_310 [[deprecated("This will be removed in SeqAn-3.1.0; please see the documentation.")]]
 #endif
 
+// ============================================================================
+//  Architectures
+// ============================================================================
+
+// Warn if NO_WARN_X86_INTRINSICS is not set on PowerPC. See https://github.com/seqan/seqan3/pull/1157.
+#if defined(__powerpc64__)
+#   ifndef NO_WARN_X86_INTRINSICS
+#      warning "To use SeqAn3 on PowerPC, you may need to set -DNO_WARN_X86_INTRINSICS. Please note that SeqAn3 is not yet optimised for PowerPC."
```

Comment:
Line wrap?

Reply:
Ok, I changed it.

Looks now like this:
```console
/Users/enricoseiler/git/seqan3/include/seqan3/core/platform.hpp:199:28: error: To use SeqAn3 on PowerPC, you may need to set -DNO_WARN_X86_INTRINSICS.                             Please note that SeqAn3 is not yet optimised for PowerPC. [-Werror]
 #       pragma GCC warning "To use SeqAn3 on PowerPC, you may need to set -DNO_WARN_X86_INTRINSICS. \
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                             Please note that SeqAn3 is not yet optimised for PowerPC."
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

(Not that there is no way to do a linebreak AFAIK, you can use `\` to make the preprocessor evaluate the next line in the current expression, but you can't escape the whitespaces used for indentation.)

But I think it looks fine now.

Reply:
But does it work? I mean, if they are x86-intrinsics (popcount comes to mind) what happens when they are used on PPC?

Reply:
Tests work fine, except the one I wrote about above.

Internally, if an explicit intel intrinsic pops up, the behaviour is emulated on the PPC (also see the header I added in the first post). This may be a bit slower, but is well defined.